spec.html

```diff
@@ -454,18 +625,19 @@ <h1>ShadowRealm.prototype.evaluate ( _sourceText_ )</h1>
 			</emu-note>
 		</emu-clause>
 
-		<emu-clause id="sec-shadowrealm.prototype.importvalue">
-			<h1>ShadowRealm.prototype.importValue ( _specifier_, _exportName_ )</h1>
+		<emu-clause id="sec-shadowrealm.prototype.import">
+			<h1>ShadowRealm.prototype.import ( _specifier_ )</h1>
 			<p>The following steps are performed:</p>
 			<emu-alg>
 				1. Let _O_ be *this* value.
 				1. Perform ? ValidateShadowRealmObject(_O_).
+				1. Let _promiseCapability_ be ! NewPromiseCapability(%Promise%).
 				1. Let _specifierString_ be ? ToString(_specifier_).
-				1. If Type(_exportName_) is not String, throw a *TypeError* exception.
 				1. Let _callerRealm_ be the current Realm Record.
 				1. Let _evalRealm_ be _O_.[[ShadowRealm]].
 				1. Let _evalContext_ be _O_.[[ExecutionContext]].
-				1. Return ? ShadowRealmImportValue(_specifierString_, _exportName_, _callerRealm_, _evalRealm_, _evalContext_).
+				1. Perform HostShadowRealmImportModuleDynamically(_specifierString_, _promiseCapability_, _callerRealm_, _evalRealm_, _evalContext_).
+				1. Return _promiseCapability_.[[Promise]].
```

Comment:
Nit pick: We should add an _editorial note_ saying this method is designed to eventually house a parameter for import assertions.